src/twister2/helper.py

```diff
@@ -1,5 +1,11 @@
 from __future__ import annotations
 
+import logging
+import platform
+import shlex
+
+_WINDOWS = (platform.system() == 'Windows')
```

Comment:
In v1  checks for os like `if (os.name != "nt")` were used. Are there benefits of platform over os? Maybe it'll be better to stick with one option.

Reply:
`os.name` isn't always used in v1. For example:
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib/log_helper.py#L11

After analyzing following discussions:
https://stackoverflow.com/questions/4553129/when-to-use-os-name-sys-platform-or-platform-system
https://www.quora.com/Whats-the-difference-between-platform-system-Windows-and-OS-name-nt
https://docs.python.org/3/library/platform.html#platform.system
I can say, that there is no big difference between those two approaches (`platform.system()` and `os.name`). `platform.system()` can help you distinguish exactly what distribution of OS you have ('Linux', 'Darwin', 'Java', 'Windows'). In this particular situation in this PR this information is unnecessary.

In my own opinion we can stay with `platform.system() == 'Windows'` - for me it is more readable than `os.name != "nt"`.